_includes/1.0/api/json/quote.json

```diff
@@ -0,0 +1,64 @@
+{
+  "id": 10004,
+  "status": "open",
+  "customer_id": 10000,
+  "billing_contact_id": 10000,
+  "billing_address": {
+    "street_line1": "2 Rue de Rivoli",
+    "street_line2": "",
+    "street_line3": "",
+    "postal_code": "75004",
+    "city": "Paris",
+    "country": "France",
+    "country_iso_code": "FR",
+    "locality": "",
+    "short_locality": "",
+    "full_address": "2 Rue de Rivoli 75004 Paris France"
+  },
+  "po_number": null,
+  "tax_id": null,
+  "reduce_rate": 20.0,
+  "order_lines": [
+    {
+      "id": 10002,
+      "item_id": 10000,
+      "description": "A service",
+      "quantity": 1.0,
+      "unit_price_wt": 1000.0,
+      "discount_rate": 0.0,
+      "tax_id": 1,
+      "is_taxable": false,
+      "total": 1000.0
+    }
+  ],
+  "public_note": "Note to print",
+  "shipping_date": null,
+  "shipping_contact_id": 10000,
+  "shipping_address": {
+    "street_line1": "2 Rue de Rivoli",
+    "street_line2": "",
+    "street_line3": "",
+    "postal_code": "75004",
+    "city": "Paris",
+    "country": "France",
+    "country_iso_code": "FR",
+    "locality": "",
+    "short_locality": "",
+    "full_address": "2 Rue de Rivoli 75004 Paris France"
+  },
+  "payment_terms_id": 1,
+  "project_id": 10000,
+  "delivery_status": null,
+  "invoice_status": null,
```

Comment:
And then if you convert the Delivery Note into an Invoice, you should have status here